src/assets/HomepageModal.scss

```diff
@@ -0,0 +1,28 @@
+#landing-page-cards {
```

Comment:
Is there any reason to have `css` and `scss` files? 
Can we use only 1 type

Reply:
I don't have a problem with that, I was building off of what others had done, and I think @Kachulio1 added it as `scss` because he was using a variable for border radius, but I removed it.

Actually, we need to have a better look at the styling cause I was looking at it with @javpet and it seems like @dkuku set it up to use variables, but with `less`(?)